nautobot/docs/models/extras/relationship.md

```diff
@@ -9,11 +9,40 @@ Sometimes it is desirable to create a new kind of relationship between one (or m
 
 ## Relationship Filters
 
-Filters can be defined to restrict the type or selection of objects for either side of the connection. From the FEX example above, you can restrict the FEX side of the connection to only of devices with the FEX role, and restrict the controller side to only devices with Controller roles.
+Filters can be defined to restrict the type or selection of objects for either side of the connection. 
+
+### Relationship Filter Example
+
+As an example, let's create a relationship between Circuits and Devices. 
+In our situation we only would terminate Circuits on Devices with the Role of `edge`.
+
+To prevent the Circuit Relationship from showing up on any other Device, use a JSON filter to 
+limit the Relationship to only Devices with Role of `edge`:
```

Comment:
```suggestion
As an example, let's create a relationship between Circuits and Devices. 
In our situation we only would terminate Circuits on Devices with the Device Role of `edge`.

To prevent the Circuit Relationship from showing up on any other Device, use a JSON filter to 
limit the Relationship to only Devices with a Device Role whose slug is `"edge"`:
```